semanticcpg/src/main/scala/io/shiftleft/semanticcpg/language/package.scala

```diff
@@ -219,13 +219,12 @@ package object language {
   implicit def toNodeTypeStartersOps(cpg: Cpg): OpNodeTypeStarters =
     new OpNodeTypeStarters(cpg)
 
-  implicit def arrayAccessToCall(
-      node: operatorextension.nodes.ArrayAccess): io.shiftleft.semanticcpg.language.nodemethods.CallMethods =
-    node.call
+  implicit def toOpsAstNodeTrav2[NodeType <: nodes.AstNode](
```

Comment:
Might be worth dropping the `2` in the name, I'm not too sure what it means here!